internal/cni-server/cni-plugin.go

```diff
@@ -256,79 +257,83 @@ func (s *server) listenConfig(addr net.Addr, netns string) net.ListenConfig {
 	}
 }
 
-// func (s *server) checkExistingPods() error {
-// 	dir, err := os.ReadDir("/host/proc")
-// 	if err != nil {
-// 		return err
-// 	}
-// 	for _, f := range dir {
-// 		if _, err = strconv.Atoi(f.Name()); err == nil {
-// 			netns, err := ns.GetNS(fmt.Sprintf("/host/proc/%s/ns/net", f.Name()))
-// 			if err != nil {
-// 				log.Errorf("Failed to get ns for %s", netns.Path())
-// 				continue
-// 			}
-// 			ignored := false
-// 			if err = netns.Do(func(_ ns.NetNS) error {
-// 				if ignore() {
-// 					// ignore uninjected pods
-// 					ignored = true
-// 					return nil
-// 				}
-// 				log.Infof("build listener for pid(%s)", f.Name())
-// 				// listen on 39807
-// 				return s.buildListener(netns.Path())
-// 			}); err != nil {
-// 				return err
-// 			}
-// 			if !ignored {
-// 				// attach xdp to the veth pair
-// 				if err = netnsPairEthDo(netns.Path(), func(name string, index int) error {
-// 					xdp, err := ebpf.LoadPinnedProgram(path.Join(s.bpfMountPath, "mb_xdp"), &ebpf.LoadPinOptions{})
-// 					// todo support load by ID: xdp, err := ebpf.NewProgramFromID(1595)
-// 					if err != nil {
-// 						log.Errorf("Failed to load %s", path.Join(s.bpfMountPath, "mb_xdp"))
-// 						return err
-// 					}
+func (s *server) checkExistingPods() error {
+	hostProc, err := os.ReadDir(config.HostProc)
+	if err != nil {
+		return err
+	}
+	for _, f := range hostProc {
```

Comment:
Scanning all process is too expensive for us, we may need to find a more stable way.
But it doesn't need to be modified now, I'm just reminding.